thorcast/geocode.py

```diff
@@ -1,7 +1,21 @@
+import os
+
 import requests
 
 
-def geocode(city, state, apikey):
+def geocode(city, state):
```

Comment:
same naming critique here.
```python
import geocode

result = geocode.fetch(city, state)
# is better than
result = geocode.geocode(city, state)
```
I don't know if I'm going against the PEP grain here, but I really don't like when projects do:

```python
from geocode import geocode
result = geocode(city, state)
# or
from geocode import geocode as g
result = g(city, state)
```

Reply:
I agree, didn't really think of it though.